packages/create-contentful-studio-experiences/package.json

```diff
@@ -1,9 +1,8 @@
 {
-  "name": "@contentful/create-experience-builder",
-  "private": true,
```

Comment:
we might want to keep this private on to keep the package from publishing until its ready to go

Reply:
Ok sounds good, I'll set it back to private in the next commit